lnrpc/wtclientrpc/wtclient.proto

```diff
@@ -88,6 +96,14 @@ message RemoveTowerRequest {
 message RemoveTowerResponse {
 }
 
+message DeactivateTowerRequest {
+    // The identifying public key of the watchtower to deactivate.
+    bytes pubkey = 1;
+}
+
+message DeactivateTowerResponse {
```

Comment:
Empty response messages on the CLI are confusing most users... Should we add a simple status as I'm proposing [here](https://github.com/lightningnetwork/lnd/pull/7762)?